internal/repository/sellers/create_test.go

```diff
@@ -0,0 +1,116 @@
+package sellersrp
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/go-sql-driver/mysql"
+	"github.com/maxwelbm/alkemy-g6/internal/factories"
+	"github.com/maxwelbm/alkemy-g6/internal/models"
+	"github.com/maxwelbm/alkemy-g6/pkg/mysqlerr"
+	"github.com/maxwelbm/alkemy-g6/pkg/testdb"
+	"github.com/stretchr/testify/require"
+)
+
+func TestCreate(t *testing.T) {
+	db, truncate, teardown := testdb.NewConn(t)
+	defer teardown()
+	factory := factories.NewSellerFactory(db)
+	fixture := factory.Build(models.Seller{ID: 1})
+
+	type arg struct {
+		dto models.SellerDTO
+	}
+	type want struct {
+		seller models.Seller
+		err    error
+	}
+	tests := []struct {
+		name  string
+		setup func()
+		arg
+		want
+	}{
+		{
+			name: "When create a new Seller",
+			setup: func() {
+				err := factory.CheckLocalityExists(fixture.LocalityID)
+				require.NoError(t, err)
```

Comment:
se fizer assim nao precisa deixar aquele método publico
```suggestion
			setup: func() {
				_, err := factories.NewLocalityFactory(db).Create(models.Locality{ID: 1})
				require.NoError(t, err)
```

Reply:
👍